### ISSUE TITLE
OneVsOneClassifier Support

### ISSUE THREAD

**Issue by nareshshah139** [TeamHG-Memex/eli5#226]
Looks like ELI5 does not work with Multi-class classifiers in sklearn though it works with the same in xgboost. Perhaps this can be a useful new feature.

**Comment by kmike** [comment]
Yeah, currently we support sklearn multi-class classifiers which use OvR strategy (internally or with OneVsRestClassifier wrapper); OvO is not yet supported. 

For example, if you use RandomForestClassifeir without OvO wrapper, explanations would work (as RandomForestClassifeir supports multi-class natively).

**Comment by nareshshah139** [comment]
The reason I ask this is the performance differs between multi-class RandomForest and OvO RandomForest/OvR Random Forest as well. It is puzzling though as to why there is a significant difference in the first place. That is something I was trying to debug.
